instrumentation/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+"""
+This library contains all android_binary targets to each test suite in instrumentation module.
+Note that:
+ - All the android_binaries are named similar to the respective class name of the test suite.
+"""
+
+android_binary(
+    name = "BaseTest",
```

Comment:
I decided to use the class name as the name for the android_binary so It might be easy when install the test app